appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -1890,6 +1890,13 @@
 (define (string-to-lower-case s)
   (String:toLowerCase (s:toString)))
 
+(define (string-to-title-case s)
```

Comment:
This only capitalizes the first letter of the string. Title case would be capitalizing the first letter of every word.  Also, the way this is written if the string is empty then this function returns nothing.

Reply:
+ okay that was a big blunder from my side in understanding the use-case. I will change my logic for it to capitalize first letter of every word. 
+ I will also return an empty string if it gets and empty string

Reply:
@ewpatton can you suggest me some way to test the code that I have written in runtime.scm because right now I don't have a proper grasp at functional programming languages. so I have to build an apk every time I want to test anything in the runtime.scm

Reply:
You can use kawa interactively at the command line to test your scheme code by running:

```
java -jar lib/kawa/kawa-1.11-modified.jar
```

An alternative approach is to implement the code in Java and then call through from Scheme to  Java, such as in #2085.